app/models/category_promo.rb

```diff
@@ -0,0 +1,5 @@
+class CategoryPromo < ActiveRecord::Base
+  PROMO_TYPES = %w{ blog recommended popular calculator tool }
+
+  belongs_to :category, class: Comfy::Cms::Category
+end
```

Comment:
I know your protecting against submissions with no title but shouldn't that be backed up with validations in here?

Same for ensuring promo type is included in PROMO_TYPES.